test/extended/util/url/url.go

```diff
@@ -272,7 +342,24 @@ func (ut *Test) ToShell(i int) string {
 		}
 	}
 	lines = append(lines, `rc=0`)
-	cmd := fmt.Sprintf(`curl -X %s %s -s -S -o /tmp/body -D /tmp/headers %q`, ut.Req.Method, strings.Join(headers, " "), ut.Req.URL)
+	post := ""
+	if strings.ToLower(strings.Trim(ut.Req.Method, " ")) == "post" {
+		post = " -H 'Expect:' "
+		if len(ut.PostBodyFile) > 0 {
+			basename := filepath.Base(ut.PostBodyFile)
+			dirname := filepath.Dir(ut.PostBodyFile)
+			lastsubdir := filepath.Base(dirname)
+			err := ut.Oc.AsAdmin().Run("rsync").Args(dirname, ut.PodName+":"+"/tmp", "--strategy=tar").Execute()
+			if err != nil {
+				e2e.Logf("errror on rsync: %#v", err)
+			} else {
+				post = post + " -d @/tmp/" + lastsubdir + "/" + basename
```

Comment:
not clear the implications of this... if the rsync fails, we assume the data file exists inside the pod image at this path?